### DISCUSSION TITLE
Nginx Proxy Manager LXC script failed

### DISCUSSION THREAD

**Discussion by yuriw** [tteck/Proxmox#2311] [Q&A] [answered]
At the end of the installation I got an error `- Installing Node.js  //root/.bashrc: line 6: PS1: unbound variable`

Any clues on how to workaround?

TIA

**Comment by yuriw** [comment]
I used advanced options and only changed CT name to 'nginx', ubuntu 22.04 and root password for ssh 

Installation with default options worked fine.  I don't know what is the default user/password for ssh?

**Reply by tteck** [reply]
> - If the default Linux distribution is not adhered to, script support will be discontinued.

**Reply by yuriw** [reply]
Are you suggesting that Ubuntu 22.04 is not supported?

What's is default user password for ssh if default options were used?

Or ssh then disabled?

Thx

**Reply by tteck** [reply]
It states **default Linux distribution** NOT default settings.

**Reply by yuriw** [reply]
I am not sure what you are saying.
Can you please elaborate and help with my issues?

**Reply by tteck** [reply, marked as answer]
You're free to customize your LXC creation as you see fit. However, if you opt for a different distribution than the default one, which is **Debian 12** in this instance, and encounter any issues, you'll need to address and resolve them on your own.

**Comment by AlexGeek77** [comment]
> I used advanced options and only changed CT name to 'nginx', ubuntu 22.04 and root password for ssh
> 
> Installation with default options worked fine. I don't know what is the default user/password for ssh?

During the install script you're presented with this screen:
<img width="777" alt="Screenshot 2024-01-09 at 1 36 12 PM" src="https://github.com/tteck/Proxmox/assets/1232415/38893aa6-ef77-4c1e-a61b-e67ef345aad6">

If you left it blank then you'll need to create a local user for remote access or go through the process of setting a password for root.

**Comment by yuriw** [comment]
@tteck 

I think I did not appreciate the meaning of this:

![image](https://github.com/tteck/Proxmox/assets/3876474/9db55d5f-8c3e-484a-a86d-6a94bf609cfe)

In fact, it means that the script was tested and developed only for Debian.

Maybe it would benefit from stronger wording.
Thx for clarification.
